src/clock-weather-card.ts

```diff
@@ -440,19 +441,13 @@ export class ClockWeatherCard extends LitElement {
   }
 
   private time (date: DateTime = this.currentDate): string {
+    // Honor deprecated time_format setting until removed
```

Comment:
in order not to introduce a breaking change, the precedence should be `time_pattern` -> `time_format` -> `default`, so please do not remove existing code, but add 

```
if (this.config.time_pattern) {
   return this.toZonedDate(date).toFormat(this.config.time_pattern)
}
```

as the first condition in function `time(...)`